src/Util/Crypto.php

```diff
@@ -336,25 +336,25 @@ public static function encapsulate($data, $pubkey, $alg = 'aes256cbc')
 	}
 
 	/**
-	 * 
+	 *
 	 * Ported from Hubzilla: https://framagit.org/hubzilla/core/blob/master/include/crypto.php
-	 * 
+	 *
 	 * @param type $data
 	 * @param type $pubkey The public key.
-	 * @param type $alg    The algorithm used for encryption.
```

Comment:
Those white spaces are actually wanted, so all the parameter comments are aligned.